core/src/main/java/org/apache/gravitino/storage/relational/JDBCBackend.java

```diff
@@ -382,20 +380,23 @@ public <E extends Entity & HasIdentifier> List<E> listEntitiesByRelation(
       case METADATA_OBJECT_ROLE_REL:
         return (List<E>)
             RoleMetaService.getInstance()
-                .listRolesByMetadataObjectIdentAndType(nameIdentifier, identType);
+                .listRolesByMetadataObjectIdentAndType(nameIdentifier, identType, allFields);
       case ROLE_GROUP_REL:
         if (identType == Entity.EntityType.ROLE) {
           return (List<E>) GroupMetaService.getInstance().listGroupsByRoleIdent(nameIdentifier);
         } else {
           throw new IllegalArgumentException(
-              String.format("ROLE_GROUP_REL doesn't support type %s", identType.name()));
+              String.format(
+                  "ROLE_GROUP_REL doesn't support type %s or loading all fields",
```

Comment:
Is it possible that ROLE_GROUP_REL does not support all fields?

Reply:
Now, we don't support all fields.

Reply:
What is the meaning of this error message, I cannot quite follow your meaning.

Reply:
Fixed the error message, remove wrong message `or loading all fields`.